### ISSUE TITLE
Getting Prettier error when updating mui-public and mui-private to the latest Toolpad version

### ISSUE THREAD

**Issue by prakhargupta1** [mui/mui-toolpad#2808]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Steps to reproduce 🕹

I tried updating the Toolpad version in [tools-private](https://github.com/mui/mui-private/tree/master/tools-private) and [tools-public](https://github.com/mui/mui-public/tree/master/tools-public) but was getting a prettier.config error:

<img width="1448" alt="Screenshot 2023-10-16 at 7 22 37 PM" src="https://github.com/mui/mui-toolpad/assets/92228082/1171e7e1-1c35-40f1-bb04-b1725e4fda50">

### Your environment 🌎

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
  Don't forget to mention which browser you used.
  Output from `npx @mui/envinfo` goes here.
```
</details>

**Comment by apedroferreira** [comment]
I can't reproduce this on `mui-private` or `mui-public`, Toolpad version `0.1.32`.

I think it might an issue with the repos themselves in your local environment - have you tried running `yarn install` both on the root project and in `tools-private`? They are separate things in these projects.

If you still get errors after that I did have some issues that I ended fixing by first deleting all `node_modules` folders and `yarn.lock` files everywhere and then running the `yarn install`s as above.
